packages/shared/lib/index.ts

```diff
@@ -33,6 +33,7 @@ export * from './services/invitations.js';
 export * from './services/providers.js';
 
 export * as oauth2Client from './clients/oauth2.client.js';
+export * from './clients/locking.js';
```

Comment:
I should have merged my PR before #2789 😭

Reply:
ah yes sorry about that